frontend/src/app/shared/components/challenges-dropdown-select/challenges-dropdown-select.component.html

```diff
@@ -0,0 +1,24 @@
+<div ngbDropdown class="dropdown d-inline-flex" [autoClose]="true">
```

Comment:
if this component is specific to challenges and only is used in this module that they should be a part of it

Reply:
Oh, that is not mine component, in suggested challenge component filters I use only ones that was written before, this dropdown is from Sasha Bondarchuk task 57 challenges test page,  I merged his branch into mine just in case if this task would require custom icons on dropdown element, (but it was highlighted in main page bug), the reason behind was to sync your changes so we will not have the same implementation of the same thing, I've done it after we discussed how to deal with this situation and came to this conclusion, should I revert that merge, wait for his component to be finished or just leave it as it is?

Reply:
Probably it is depends on what PR will merged faster, so for now we are good with this